DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/DependencyInjection/AutofacArcGISModule.cs

```diff
@@ -37,8 +40,16 @@ protected override void Load(ContainerBuilder builder)
     builder.RegisterType<AccountBinding>().As<IBinding>().SingleInstance();
     builder.RegisterType<BasicConnectorBinding>().As<IBinding>().As<IBasicConnectorBinding>().SingleInstance();
     builder.RegisterType<ArcGISSendBinding>().As<IBinding>().SingleInstance();
+    builder.RegisterType<ArcGISReceiveBinding>().As<IBinding>().SingleInstance();
     builder.RegisterType<ArcGISToSpeckleUnitConverter>().As<IHostToSpeckleUnitConverter<Unit>>().SingleInstance();
 
+    // Operations
+    builder.RegisterType<ReceiveOperation>().AsSelf().SingleInstance();
```

Comment:
This will need to be InstancePerLifetimeScope, you'll need to ensure anything that is injected into an UoW is InstancePerLifetimeScope typically, though there *may* be the odd case where this can SingleInstance()

Reply:
(it will also need interfacing out :D )